### PR TITLE
Added cancer symbol

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
Added cancer symbol

### CODE REVIEW

**Comment by Pradumnasaraf** [comment]
@DenverCoder1 plz check

File: dictionary.txt [hunk 1]
```diff
@@ -883,4 +883,5 @@ _rp	₎
 \ddagger	‡	
 \smalltilde	˜	
 -	−	
-\taurus	♉	
\ No newline at end of file
+\taurus	♉	
+\cancer	♋
```
Comment (DenverCoder1) [inline dictionary.txt:887]:
```suggestion
\taurus	♉	
\cancer	♋	
```

**Comment by Pradumnasaraf** [comment]
done

**Comment by Pradumnasaraf** [comment]
@DenverCoder1 plz check

**Comment by Pradumnasaraf** [comment]
Thank you for the merge :), you really help me a lot

**Pull request merged**